src/js/components/agency/v2/CountTab.jsx

```diff
@@ -13,13 +13,15 @@ const propTypes = {
     setActiveTab: PropTypes.func.isRequired,
     active: PropTypes.bool,
     count: PropTypes.number,
-    subCount: PropTypes.number
+    subCount: PropTypes.number,
+    disabled: PropTypes.bool
 };
 
 const CountTab = (props) => (
     <button
         className={`count-tabs__button${props.active ? ' count-tabs__button_active' : ''}`}
-        onClick={() => props.setActiveTab(props.type)}>
+        onClick={() => props.setActiveTab(props.type)}
```

Comment:
We should be avoiding binding and arrow function in render whenever possible from React docs `Using an arrow function in render creates a new function each time the component renders, which may break optimizations based on strict identity comparison.` ( https://reactjs.org/docs/faq-functions.html ).  We can move this function out to be `const setActiveTab = (e) => props.setActiveTab(e.target.value)` and pass a `value` prop to the `button` by `value={props.type}`

Reply:
Refactored.